web/src/app/app.component.ts

```diff
@@ -79,6 +81,11 @@ export class AppComponent implements OnInit, OnDestroy {
     });
   }
 
+  logout() {
+    this.taskanaEngineService.logout().subscribe(() => {});
```

Comment:
empty subscription? Did you mean write the 86 line inside this?

Reply:
I actually didn't change the function, only moved it from the old nav-bar component to the app component. So I don't even know why there is a subscription needed. I guess I can just remove it, shouldn't change anything.

Reply:
👍